tests/www/test_utils.py

```diff
@@ -17,12 +17,19 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import unittest
+from bs4 import BeautifulSoup
 import mock
-from xml.dom import minidom
+import six
+from six.moves.urllib.parse import parse_qs
 
 from airflow.www import utils
 
+if six.PY2:
+    # Need `assertWarns` back-ported from unittest2
```

Comment:
@ashb "assertWarns" -> "assertRegex"?

Reply:
Copy-paste fail! Fixed, thanks

Reply:
Hi @ashb , seems you missed to change this earlier? It's very minor though